pkgs/development/libraries/aqbanking/sources.nix

```diff
@@ -1,12 +1,11 @@
-# This file is autogenerated from update.sh in the same directory.
```

Comment:
Is it intended that the `update.sh` script is removed below?

If yes, it would be good to write the rationale into the commit message (I'm not familiar with `aqbanking` but I'd imagine that script was for convenience to not have to figure out the versions by hand).